Analogies/Analogy_Functions.R

```diff
@@ -0,0 +1,203 @@
+## Analogies App
+## Alexandra Pafford
+
+setwd("C:/Users/alexa/Documents/Programming2/Analogies/R")
+getwd()
+
+# --------------------------------------------------------------------------
+
+## Install dependencies
+# install.packages(c("shiny", "readr", "writexl"))
+
+## Load libraries with namespaces
+library(shiny)
+library(readr)
+library(writexl)
+
+# --------------------------------------------------------------------------
+## Functions
+
+#' @title Generate Sequential String of Letters
+#' @description Generates a sequential string of a specified length starting from a random letter.
+#' @param length The length of the letter string to generate.
+#' @return A string of sequential letters of the specified length.
+#' @examples
+#' generate_sequential_string(5)
```

Comment:
I'm not sure what "sequential string of letters" means, I'm guessing something like "hijkl"? it would be helpful to have this clearer in the function documentation. While the example you put in lets someone run it to see what it does, I think also adding in what kind of return you can expect (like you do with the next function) adds some convenience. This would also clarify that you're generating capitalised letters.

Reply:
Great suggestion, thank you